indico/modules/events/operations.py

```diff
@@ -332,7 +327,8 @@ def update_event_protection(event, data):
                       'access_key': {'title': 'Access key', 'type': 'string'},
                       'visibility': {'title': 'Visibility', 'type': 'string',
                                      'convert': lambda changes: [format_visibility(event, x) for x in changes]},
-                      'public_regform_access': 'Public registration form access'}
+                      'public_regform_access': 'Public registration form access',
+                      **(_extra_log_fields)}
```

Comment:
```suggestion
                      **(_extra_log_fields or {})}
```

otherwise it fails if the arg is omitted